tests/data/non-default_folders/def/region/regions.yaml

```diff
@@ -0,0 +1,8 @@
+- common:
+    - World
+- country:
+    - Austria:
+        iso2: AT
```

Comment:
While technically correct, these lines aren't strictly needed for the unit tests to work so I would remove them and just leave the `Austria` entry.